pkgs/top-level/all-packages.nix

```
@@ -8567,7 +8567,7 @@ in
 
   rpi-imager = libsForQt5.callPackage ../tools/misc/rpi-imager { };
 
-  rpiboot-unstable = callPackage ../development/misc/rpiboot/unstable.nix { };
+  rpiboot = callPackage ../development/misc/rpiboot/unstable.nix { };
```

Maybe we can rename `pkgs/development/misc/rpiboot/unstable.nix` to `pkgs/development/misc/rpiboot/default.nix` too?

```suggestion
  rpiboot = callPackage ../development/misc/rpiboot { };
```

---

Done, didn't notice the filename while editing :)